### PR TITLE
Menu mobile

### PR DESCRIPTION
**Pull request by rebel216**
Added Mobile menu support using Javascript eventListners and Queryselector.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,10 +14,32 @@
...
+            <li>
+              <a href="#aboutme-id" class="link-about">
+                About
+              </a>
+            </li>
+            <li>
+              <a href="#footer-contact-id" class="link-contact">
+                Contact
+              </a>
+            </li>
```
Comment (ShadyShawkat) [inline index.html:40]: - [ ] Please make sure to implement the same functionality of the mobile menu nav links to the desktop nav links.
- [ ] Kindly make sure that the close button is visible on mobile view, (See below image)
image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (rebel216) [reply index.html:40]: Done as required .. Thank You

**Review by KayLemba — APPROVED** [review]
Hi TEAM
Good job!!! Your project is complete! There is nothing else to say other than... it's time to merge it 💯 
Congratulations! 🎉 



Cheers and Happy coding! 👏🏾 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**